Arashayo/TabletopHaven
# Fix filtering, fix displaying groups in profile, block non owners from adding to group,

## Related Issue
https://app.clickup.com/t/86bxnh53f
https://app.clickup.com/t/86bxnh57a

## Changes list

- fixed games filtering bug in group view
- changed displaying groups, to show only groups that logged in users own, and which user is not a member of
- prevented non owner members from being able to add to group
- removed description from groups

## Testing instructions

1. run `node install-with-users.js 50`
2. run `node run.js`
3. try to create a new group, check if it works without description
4. go to group view, check if filtering works
5. go to someones profile, check if only available groups are displaying


## Screenshots (if appropriate):
